src/helper.js

```diff
@@ -96,6 +98,14 @@ function getStoragePath() {
 	return path.join(this.HOME, "storage");
 }
 
+function getPackagesPath() {
+	return path.join(this.HOME, "packages", "node_modules");
```

Comment:
Meeeh, kinda sucks that we can't install in `packages/` itself and need a `packages/node_modules/` but oh well, looks like there is no way to do otherwise...

Reply:
Yeah, it shouldn't realistically matter because users won't be going directly into this folder, so it doesn't matter how it looks. I get your frustration, but it's really not a massive issue.